.env.example

```diff
@@ -2,4 +2,5 @@ REACT_APP_FIREBASE_API_KEY=AIzaSyAbqqXtHNIuNrsamkCxRk9sOuMO-ZWDiEk
 REACT_APP_FIREBASE_AUTH_DOMAIN=codeforcauseorg.firebaseapp.com
 REACT_APP_APP_VERSION=2.0.0
 REACT_APP_API_BASE_URL=/api
-REACT_APP_ENABLE_REDUX_LOGGER=false
\ No newline at end of file
+REACT_APP_ENABLE_REDUX_LOGGER=false
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Is it being added by mistake?

Reply:
No, 
Well when i was trying to set up the project locally, after installing dependencies when i tried to use `yarn start`, It gave the following error.

!(Error message from terminal)[https://i.imgur.com/oEoqLkB.png]

As given here the easiest way to get around this was to add ` SKIP_PREFLIGHT_CHECK=true` in .env file.

There might be a problem with the project dependency tree and likely not a bug in Create React App, but something we need to fix locally.